concordium-std/src/impls.rs

```diff
@@ -2083,16 +2083,25 @@ where
         }
     }
 
+    /// Provide clone of [`HasStateApi`] instance and new key prefix
+    /// for any container-like type wishing to store its data on blockchain
+    ///
+    /// Contract developers can use this method to implement their own
+    /// containers
+    pub fn new_state_container(&mut self) -> (S, [u8; 8]) {
```

Comment:
I feel like this should be marked with `#[must_use]` since it has global effects.

Reply:
Sure, added